src/TorchSharp/Tensor/Tensor.cs

```diff
@@ -5239,56 +5248,58 @@ public Tensor where(Tensor condition, Tensor other)
                 return right.le(left);
             }
 
-            public static explicit operator Tensor(byte value)
+            public static implicit operator Tensor(byte value)
             {
                 return torch.tensor(value);
             }
 
-            public static explicit operator Tensor(sbyte value)
+            public static implicit operator Tensor(sbyte value)
             {
                 return torch.tensor(value);
             }
 
-            public static explicit operator Tensor(short value)
+            public static implicit operator Tensor(short value)
             {
                 return torch.tensor(value);
             }
 
-            public static explicit operator Tensor(int value)
+            public static implicit operator Tensor(int value)
             {
                 return torch.tensor(value);
             }
 
-            public static explicit operator Tensor(long value)
+            public static implicit operator Tensor(long value)
             {
                 return torch.tensor(value);
             }
 
-            public static explicit operator Tensor(float value)
+            public static implicit operator Tensor(float value)
             {
                 return torch.tensor(value);
             }
 
-            public static explicit operator Tensor(double value)
+            public static implicit operator Tensor(double value)
             {
                 return torch.tensor(value);
             }
 
-            public static explicit operator Tensor(bool value)
+            public static implicit operator Tensor(bool value)
             {
                 return torch.tensor(value);
             }
 
-            public static explicit operator Tensor((float, float) value)
+            public static implicit operator Tensor((float, float) value)
             {
                 return torch.tensor(value);
             }
 
-            public static explicit operator Tensor(System.Numerics.Complex value)
+            public static implicit operator Tensor(System.Numerics.Complex value)
             {
                 return torch.tensor(value);
             }
 
+            //public static implicit operator Tensor(Scalar value) { throw new InvalidOperationException("Conversion from Scalar to Tensor"); }
```

Comment:
Any reason to keep this commented-out version in here? You have an implementation further up.

Reply:
Nope. Thanks for catching that!